README.md

```diff
@@ -131,18 +131,28 @@ LD_PRELOAD=/opt/zlib-ng/libz.so.1.2.13.zlib-ng /usr/bin/program
 
 To install zlib-ng system-wide using cmake:
 
-```
+```sh or powershell
 cmake --build . --target install
 ```
 
 ### Configure
 
 To install zlib-ng system-wide using the configure script:
 
-```
+```sh
 make install
 ```
 
+### CPack
+
+After building the cmake project, it is possible to create an installation package using cpack:
+
+```sh or powershell
+cd build
+cpack --config CPackConfig.cmake
+cpack --config CPackSourceConfig.cmake
```

Comment:
Might want to add a comment about appending `-G RPM` or `-G DEB` to easily create installable rpm or deb packages.
We default to `TGZ`, so you could add `-G <format>` to each, and have a comment about the different possible choices (the most important ones to mention in our context are probably: TGZ, ZIP, RPM, DEB).


You could also mention the less advanced method (but easier to remember) of using make, it will only do the configured TGZ packages though:
```
make package
make package_source
```

Reply:
I have updated the instructions. Regarding the `make` instructions, I rather leave that out to not make it very long and since it is very limited (only does tgz). If anybody has any questions they can search how to use cpack.